egs2/librimix/enh1/conf/tuning/train_enh_codecformer.yaml

```diff
@@ -0,0 +1,56 @@
+optim: adam
+init: xavier_uniform
+max_epoch: 1
+batch_type: folded
+batch_size: 6 # batch_size 16 can be trained on 4 RTX 2080ti
+iterator_type: chunk
+chunk_length: 24000
+num_workers: 4
+optim_conf:
+    lr: 0.00015
+    weight_decay: 0
+patience: 5
+val_scheduler_criterion:
+- valid
+- loss
+best_model_criterion:
+-   - valid
+    - si_snr
+    - max
+-   - valid
+    - loss
+    - min
+keep_nbest_models: 1
+scheduler: reducelronplateau
+scheduler_conf:
+    mode: min
+    factor: 0.5
+    patience: 1
+model_conf:
+    loss_type: si_snr
+encoder: codec
+encoder_conf:
+    channel: 512
+    codec_choice: "ESPnet"
+    codec_fs: 16000 #must match the codec
+    sample_fs: 8000
+    device: "cuda:0"
+    dump_audio: True
+    checkpoint_path: "/home/jiaqi006/code/new_espnet/soundstream_model/16khz_soundstream/train.total_count.best.pth"
```

Comment:
Avoid using absolute paths that are not reusable by other users. Please also add information on how the checkpoint should be prepared/obtained.